CruCentralCoast/Community/CommunityGroups/CommunityGroupsTVC.swift

```diff
@@ -0,0 +1,119 @@
+//
+//  CommunityGroupsTVC.swift
+//  CruCentralCoast
+//
+//  Created by Landon Gerrits on 5/16/18.
+//  Copyright © 2018 Landon Gerrits. All rights reserved.
+//
+
+import UIKit
+
+class CommunityGroupsTVC: UITableViewController {
+    
+    
+    var dataArray: [MissionCellParameters] = [MissionCellParameters(titleLabel: "Oasis", date: "March 17-26", location: "TBD", description: "ACM, the world's largest educational and scientific computing society, delivers resources that advance computing as a science and a profession. ACM provides the computing field's premier Digital Library and serves its members and the computing profession with leading-edge publications, conferences, and career resources."), MissionCellParameters(titleLabel: "test2", date: "date2", location: "location2", description: "description..."), MissionCellParameters(titleLabel: "test3", date: "date3", location: "location3", description: "description..."), MissionCellParameters(titleLabel: "test4", date: "date4", location: "location4", description: "description..."), MissionCellParameters(titleLabel: "test5", date: "date5", location: "location5", description: "description...")]
+        
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        self.navigationController?.navigationBar.prefersLargeTitles = true
+
+        // Uncomment the following line to preserve selection between presentations
+        // self.clearsSelectionOnViewWillAppear = false
+
+        // Uncomment the following line to display an Edit button in the navigation bar for this view controller.
+        // self.navigationItem.rightBarButtonItem = self.editButtonItem
+        
+        let nib = UINib.init(nibName: "CommunityTableViewCell", bundle: nil)
+        self.tableView.register(nib, forCellReuseIdentifier: "comCell")
```

Comment:
`comCell` isn't a great identifier name

Reply:
weird I dont have this file in my project